aws-rds-cfn-common/src/test/java/software/amazon/rds/common/request/ValidationsTest.java

```diff
@@ -0,0 +1,95 @@
+package software.amazon.rds.common.request;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import org.junit.jupiter.api.Test;
+
+import software.amazon.cloudformation.proxy.ResourceHandlerRequest;
+
+class ValidationsTest {
+
+    @Test
+    void testValidateEmptySourceRegion() {
+        String nullRegion = null;
+        try {
+            Validations.validateSourceRegion(nullRegion);
+        } catch (RequestValidationException e) {
+            fail("Null region is valid");
+        }
+    }
+
+    @Test
+    void testValidateSourceRegion() {
+        String validRegion = "us-east-1";
+        try {
+            Validations.validateSourceRegion(validRegion);
+        } catch (RequestValidationException e) {
+            fail("Region is valid");
+        }
+    }
+
+    @Test
+    void testValidateSourceRegionInvalid() {
+        String invalidRegion = "us-east-5";
```

Comment:
nit: might be worth to use something that is not likely to become a region name, e.g. "not-a-region". There is a potential for us-east-5 region to be used by in the future.

Reply:
done.